### PR TITLE
Theater web release1

### PR DESCRIPTION
**Pull request by Mani449**
Please Review Code and approve release 1

### CODE REVIEW

**Review by saiadityaravi — APPROVED** [review]

**Review by RahulKakarla — COMMENTED** [review]
:p

**Review by RahulKakarla — COMMENTED** [review]
i didn't do it

**Comment by AnishUnigo** [comment]
need to review

File: HelloWorld.txt [hunk 1]
```diff
@@ -4,4 +4,14 @@ Hello Anand. How are you? Version 1
 
 Hello Anand. How are you? Version 2
 
-Wow Adbutam
\ No newline at end of file
+Wow Adbutam
```
Comment (RahulKakarla) [inline HelloWorld.txt:7]: _|_

**Review by cnadella24 — APPROVED** [review]

**Pull request merged**